docs/guide/why.md

```diff
@@ -18,7 +18,7 @@ Vite improves the dev server start time by first dividing the modules in an appl
 
 - **Dependencies** are mostly plain JavaScript that do not change often during development. Some large dependencies (e.g. component libraries with hundreds of modules) are also quite expensive to process. Dependencies may also be shipped in various module formats (e.g. ESM or CommonJS).
 
-  Vite [pre-bundles dependencies](./dep-pre-bundling) using [esbuild](https://esbuild.github.io/). esbuild is written in Go and pre-bundles dependencies 10-100x faster than JavaScript-based bundlers.
+  Vite [pre-bundles dependencies](./dep-pre-bundling) using [esbuild](https://esbuild.github.io/). esbuild is written in Go and pre-bundles dependencies are 10-100x faster than JavaScript-based bundlers.
```

Comment:
The original text is grammatically correct, and your change would be incorrect.

The second clause of the sentence is:

> ebuild pre-bundles dependencies 10-100x faster than JavaScript-based bundlers.

Inserting `are`, as you suggest, would not make sense.